pkg/services/store/entity/migrations/entity_store_mig.go

```diff
@@ -120,7 +120,11 @@ func initEntityTables(mg *migrator.Migrator) string {
 		},
 		Indices: []*migrator.Index{
 			{Cols: []string{"guid", "resource_version"}, Type: migrator.UniqueIndex},
-			{Cols: []string{"namespace", "group", "resource", "name", "resource_version"}, Type: migrator.UniqueIndex},
+			{
+				Cols: []string{"namespace", "group", "resource", "name", "resource_version"},
+				Type: migrator.UniqueIndex,
+				Name: "UQE_entity_history_namespace_group_name_version",
```

Comment:
Do we not want to include "resource" as part of the index name?

Reply:
it doesn't actually make any difference as long as the name is unique, and not too long for mysql